docs/content/reference/install-configuration/entrypoints.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Traefik EntryPoints Documentation"
+description: "For routing and load balancing in Traefik Proxy, EntryPoints define which port will receive packets and whether in UDP or TCP. Read the technical documentation."
+---
+
+Listening for Incoming Connections/Requests
+{: .subtitle }
+
+### Configuration Example
+
+```yaml tab="File (YAML)"
+entryPoints:
+  web:
+    address: :80
+    http:
+      redirections:
+        entryPoint:
+          to: websecure
+          scheme: https
+          permanent: true
+
+  websecure:
+    address: :443
+    tls: {}
+    middlewares:
+      - auth@kubernetescrd
+      - strip@kubernetescrd
+```
+
+```yaml tab="Helm Chart Values"
+## Values file
+ports:
+  web:
+    port: :80
+  websecure:
+    port: :443
+    tls:
+      enabled: true
+    middlewares:
+      - auth@kubernetescrd
+      - strip@kubernetescrd
+additionalArguments:
+  - --entryPoints.web.http.redirections.to=websecure
+  - --entryPoints.web.http.redirections.scheme=https
+  - --entryPoints.web.http.redirections.permanent=true
+```
+
+!!! tip 
+
+      In the Helm Chart, the `entryPoints` `web` (port 80), `websecure` (port 443), `traefik` (port 9000) and `metrics` (port 9100) are created by default.
```

Comment:
Here I am wondering why entrypoint is highlighted in backticks? I do not think this is required, and maybe we should remove the backticks in this page.

```suggestion
      In the Helm Chart, the entryPoints `web` (port 80), `websecure` (port 443), `traefik` (port 9000) and `metrics` (port 9100) are created by default.
```

Reply:
I am also thinking if the word should be written entryPoint or entry point, both exist in this page? wdyt?

Reply:
i think entryPoint makes sense